assets/js/util/getFullURL.test.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * Absolute URL path getter utility function tests.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { getFullURL } from './getFullURL';
+
+describe( 'getFullURL', () => {
+	it( 'returns the URL with path', () => {
+		expect(
+			getFullURL( 'https://www.example.com', '' )
+		).toEqual( 'https://www.example.com/' );
+
+		expect(
+			getFullURL( 'https://www.example.com', '/path' )
+		).toEqual( 'https://www.example.com/path' );
+
+		expect(
+			getFullURL( 'https://www.example.com/slug/slug', '/path' )
+		).toEqual( 'https://www.example.com/path' );
+
```

Comment:
I added this test locally and it passed, which makes sense, so I added it explicitly so a URL with a port number in it is handled:

```js
expect(getFullURL( 'https://www.example.com:444/slug/slug', '/path' ) )
  .toEqual( 'https://www.example.com:444/path' );
```

Reply:
I left the console logs in there 🤦‍♂️.  Trying to do too many things at once. Doh!
Adding a port test was a really good idea.  I didn't think of that.